### ISSUE TITLE
Unable to set various user profile fields in 5.x branch

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#13702]
### Mautic Version

5.0.x series

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8

### What happened?

I came across this when testing another PR.

When I open the 5.x branch on GitPod and go to my user profile, I'm not able to click into or edit:
- First name
- Last name
- Signature
- Timezone
- Language fields.

Also, if I click on Authorized Applications I get the following:

![screenshot-mautic ddev site-2024 04 29-14_59_15](https://github.com/mautic/mautic/assets/2930593/33d18314-53e8-461a-a5c5-ca91fcd3dead)

and then miraculously, I can edit the fields again.

@andersonjeccel could this be as a result of the layout changes, perhaps?

### How can we reproduce this issue?

Step 1: Log in and edit your user account
Step 2: Try to change the fields above
Step 3: Click authorized applications
Step 4: Notice you can now click into the fields which were previously inaccessible

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by andersonjeccel** [comment]
Hm, that would be weird as I changed only the position inside the Twig template
But I’ll take a look soon
